lib/electric_slide/call_queue.rb

```diff
@@ -254,6 +254,8 @@ def call_agent(agent, queued_call)
       agent_call[:agent]  = agent
       agent_call[:queued_call] = queued_call
 
+      agent.call = agent_call if agent.respond_to?(:call=)
```

Comment:
@benlangfeld My understanding of e4bd696f0c was that `#call` is special to the bridge strategy, so call strategy queues may still use agents without a `#call` attribute. I can remove the conditional if it's not needed.